lighthouse-viewer/app/index.html

```diff
@@ -43,5 +43,14 @@
 </div>
 
 <script src="src/main.js" async></script>
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-85519014-2', 'auto');
+  ga('send', 'pageview');
```

Comment:
Filtering them out at the report level. We'll never see them:

<img width="416" alt="screen shot 2016-12-07 at 2 34 44 pm" src="https://cloud.githubusercontent.com/assets/238208/20989772/c65c1c00-bc8a-11e6-9279-b9bf84b2a268.png">

<img width="852" alt="screen shot 2016-12-07 at 2 37 36 pm" src="https://cloud.githubusercontent.com/assets/238208/20989780/ca9564e8-bc8a-11e6-9377-4c48305cc912.png">